spec.bs

```diff
@@ -602,7 +605,10 @@ authentication:
     transaction.
 * The {{CollectedClientAdditionalPaymentData/instrument}} details, that is the
     payment instrument {{PaymentCredentialInstrument/displayName}} and
-    {{PaymentCredentialInstrument/icon}}.
+    {{PaymentCredentialInstrument/icon}}. If an image resource could not be
+    fetched or decoded from the input {{PaymentCredentialInstrument/icon}}, and
+    {{PaymentCredentialInstrument/iconMustBeShown}} is false, then the User
```

Comment:
Technically `iconMustBeShown` must be `false` here, as otherwise we cannot reach this logic. So I think we can remove the reference to it. (If we think it is a useful clarification, it can go in a `NOTE` below this line, e.g.

```
NOTE: If the icon could not be fetched or decoded, then {{PaymentCredentialInstrument/iconMustBeShown}} must be `false` here as otherwise the [[#sctn-steps-to-check-if-a-payment-can-be-made|the steps to check if a payment can be made]] would have failed.
```

Reply:
Done, made it a note. I wanted to clarify here what the user agent can do if there's no icon.